temporalcli/commands.workflow_exec.go

```diff
@@ -404,7 +411,13 @@ func (s *structuredHistoryIter) Next() (any, error) {
 	data := structuredHistoryEvent{
 		ID:   event.EventId,
 		Time: event.EventTime.AsTime().Format(time.RFC3339),
-		Type: coloredEventType(event.EventType),
+	}
+	// TODO: This is pretty wrong - the printer should be able to determine if color should be used
+	//   or not, but that would mean adding typed color information to the structured data.
```

Comment:
This is hacky, but fixing it is sort of nonobvious. We would need to stop using strings and start using something better-typed for color, so that the printer can then print what it gets from `Next` smartly.

Not sure it's worth it.

Reply:
Color is turned on/off at a global level which is how the https://pkg.go.dev/github.com/fatih/color library works (I hate globals as much as the next guy, but it's how they did it which is ok here). So calling the color calls is a no-op when it is disabled. You don't have to worry about it as a developer, use the color calls directly and just know they are not applied when color is disabled,

Reply:
Then it's not working right. I was consistently getting color escape codes inside the JSON for even types